cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/etl/api/StageContext.java

```diff
@@ -142,6 +142,13 @@
   @Nullable
   Schema getOutputSchema();
 
+  /**
+   * Sets the output schema per run level of the pipeline if its not constant.
```

Comment:
This doc seems strange. What do mean by no constant? Shouldn't we just say `Sets the output schema`? Or there are restrictions?